jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/AttachHandler.java

```diff
@@ -22,6 +22,9 @@
  *******************************************************************************/
 package openj9.internal.tools.attach.target;
 
+import static java.util.concurrent.TimeUnit.DAYS;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
```

Comment:
Please  just import `TimeUnit` and use `TimeUnit.DAYS`, etc.

Reply:
Updated